src/control-plane/backend/lambda/api/test/api/sql-builder.test.ts

```diff
@@ -13933,5 +13931,322 @@ describe('SQL Builder test', () => {
 
   });
 
+  test('schema name hard code', () => {
```

Comment:
change to `use specified schema name in generated SQL`

Reply:
fixed